### ISSUE TITLE
Imitation train time

### ISSUE THREAD

**Issue by BetterZH** [facebookresearch/r3m#22]
Hi, I execute the following command on a single A100, but the GPU utilization is only 0%~9%. And it takes 3 days to complete the 20000 steps training.
`python hydra_launcher.py hydra/launcher=local hydra/output=local env="kitchen_sdoor_open-v3" camera="left_cap2" pixel_based=true embedding=resnet50 num_demos=5 env_kwargs.load_path=r3m bc_kwargs.finetune=false proprio=9 job_name=r3m_repro seed=125`
Is such GPU utilization normal?  How long do you spend training downstream tasks?
 
Thanks

**Comment by suraj-nair-1** [comment]
That seems very slow. Are you sure pytorch is using the GPU? And you should also verify that Mujoco is using the GPU for rendering.

**Comment by suraj-nair-1** [comment]
One run should take ~2-3 hrs

**Comment by BetterZH** [comment]
> That seems very slow. Are you sure pytorch is using the GPU? And you should also verify that Mujoco is using the GPU for rendering.

Thank you. It's really a problem with the environment installation. I've solved it.
